lib/lynx/pipe/append.rb

```diff
@@ -9,7 +9,8 @@ def initialize(file)
       end
 
       def perform(command)
-        SystemOut.run("#{command} >> #{@file}")
+        output = SystemOut.run(" #{command}")
```

Comment:
nit: space

Reply:
Given that everything is working fine now (particularly with Docker), I reverted this change and am just removing support for mysql5.5 with this pr